pom.xml

```diff
@@ -34,10 +34,10 @@
         <version.edition>Enterprise</version.edition>
         <image.tag>latest</image.tag>
 
-        <dependency.alfresco-enterprise-remote-api.version>7.94</dependency.alfresco-enterprise-remote-api.version>
-        <dependency.alfresco-enterprise-repository.version>7.120</dependency.alfresco-enterprise-repository.version>
-        <dependency.alfresco-remote-api.version>7.155</dependency.alfresco-remote-api.version>
-        <dependency.alfresco-repository.version>7.218</dependency.alfresco-repository.version>
+        <dependency.alfresco-enterprise-remote-api.version>mnt-21871-6.2.n-2</dependency.alfresco-enterprise-remote-api.version>
```

Comment:
I think these should be updated?

Reply:
That's right, they will be updated once I have new releases. Can you please take a look on https://github.com/Alfresco/alfresco-repository/pull/1244 and https://github.com/Alfresco/alfresco-remote-api/pull/833 so I can proceed with the releases?